wallet/chainntfns.go

```diff
@@ -489,7 +495,7 @@ func (w *Wallet) processTransactionRecord(ctx context.Context, dbtx walletdb.Rea
 				stakemgrNs, addr, &rec.Hash)
 			if err != nil {
 				log.Warnf("Failed to update pool user %v with "+
-					"invalid ticket %v", addr.Address(),
+					"invalid ticket %v", addr,
```

Comment:
Not that it matters much, but I expect the following param could be moved up and it will fit now.